src/plugins/discover/public/application/main/utils/resolve_index_pattern.ts

```diff
@@ -7,7 +7,7 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import type { IndexPattern, IndexPatternsContract, ISearchSource } from 'src/plugins/data/common';
+import type { DataView, DataViewsContract, ISearchSource } from 'src/plugins/data/common';
```

Comment:
Should we use relative imports for plugins?

Reply:
there is no general guide for this AFAIK. for internal imports in plugins it has to be relative, but for importing across plugin both works, however I think it has advantages when refactoring using an IDE. think we should stick to one pattern and clean up the other pattern in a follow up.